server/shops.lua

```diff
@@ -81,7 +81,7 @@ QBCore.Functions.CreateCallback('jl-laptop:server:checkout', function(source, cb
     }
     if Saved then
         for _, v in pairs(Saved) do
-            Shop.items[Config[appLabel].Items[v.name].name] = v.quantity
+            Shop.items[Config[appLabel].Items[v.name]] = v.quantity
```

Comment:
Same thing here .name is needed as you are indexing a table value instead of a string value